src/datasets/utils/resources/languages.json

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
Where did you get this data from ? (for future references and eventual updates)

Reply:
I got the language codes from `datahub.io` (https://datahub.io/core/language-codes) and mapped the Language names manually with the `langcodes` package.
I have added a comment here: https://github.com/huggingface/datasets/pull/2107/files#diff-00db9a5082ea37ce6236856b9c01bdcca69facd06a0d9484141dd2c81c62f5f0R30-R32
Let me know if the statement is not clear 😄

Reply:
Ok thanks :)